docs/diagrams/UpcomingMeetingDiagrams/UpcomingMeetingActivityDiagram.puml

```diff
@@ -0,0 +1,25 @@
+@startuml
+!pragma useVerticalIf on
+
+start
+:User presses f2;
+
+:check a list of Person objects if any has meetings in the next 7 days;
+if() then ([else])
+:filtered out Person object;
+stop
+
+else ([has meetings in next 7 days])
+:check filtered list for specific meeting times in the next 7 days;
```

Comment:
is it true that it checks the list and then checks the filtered list again? just making sure its correct